config/functions.config.local.js

```diff
@@ -5,11 +5,9 @@ var config = {
   sendgrid: {
```

Comment:
right above here should update the URL to portal.covidwatch.org, right?  Or do we want to save that update for when we are ready to actually push this to prod and keep the scope of this PR more staging?  Not sure if matters either way.